account_move_journal_template/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,3 @@
+To configure this module, you need to:
+
+#. Install it.
```

Comment:
Well you need to go to the product (templates) and add a journal entry templates to them...

Reply:
OK, this is taken up in the Usage